compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STSpreadMemberNode.java

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.compiler.internal.parser.tree;
+
+import io.ballerina.compiler.syntax.tree.Node;
+import io.ballerina.compiler.syntax.tree.NonTerminalNode;
+import io.ballerina.compiler.syntax.tree.SpreadMemberNode;
+import io.ballerina.compiler.syntax.tree.SyntaxKind;
+
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * This is a generated internal syntax tree node.
+ *
+ * @since 2.0.0
```

Comment:
Shouldnt this be 2201.1.0?

Reply:
Good catch. This also has to be fixed with #31047. For now, these places were manually updated with [439ccce](https://github.com/ballerina-platform/ballerina-lang/pull/35386/commits/439cccef7ecd1572d544e7241573fe21600f0bf0).